content/post/kpis.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "KPI Principles"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["analytics", "KPI"]
+date: 2019-03-30T21:19:01-04:00
+draft: true
+---
+
+Everyone wants to know how their business is doing. They also want to know what the levers available to them are, and how to see if those levers make a difference. Enter the Key Performance Indicator (KPI). KPIs are a common tool for quantifying the performance of your business. They're also a symbol for folks in the organization to rally around and anchor against, providing clarity and aligning cross departmental objectives.
+
+<!--more-->
+
+There is a lot of content out there on what a KPI is (and what a metric is, and what a measure is, ..) and why they are important[^1]. This content is also usually written for non-analysts -- useful, but invariably leaves me wanting more. Today I'm going to outline a few key principles related to KPIs aimed specifically at analysts and data scientists. I think about KPI principles along two levels:
```

Comment:
I think I found this post a little confusing because it's unclear to me the appropriate context. Is the context "the executive team has told you, the analyst, that we need KPIs for the business so go figure it out" or is it like "you, the analyst, are creating KPIs for the analytics and data team _only_".

I _think_ you're writing this post in the context of the former, but you sort of drift between talking about KPIs in general and about specific things for analysts to do. It might be more clear if you start by setting the context and then diving into the analysts role more concretely.